### PR TITLE
Ozone guy/level2

### PR DESCRIPTION
**Pull request by OZoneGuy**
Added the second level.

### CODE REVIEW

File: Assets/Scenes/Level2.unity [hunk 1]
```diff
@@ -554,14 +554,14 @@ MonoBehaviour:
...
-  noneMessage: 
-  magniferMessage: 
-  brushMessage: 
-  materialMessage: 
-  coveredMessage: 
-  magnifierClip: {fileID: 0}
-  materialClip: {fileID: 0}
-  tipManager: {fileID: 0}
+  noneMessage: NA
+  magniferMessage: The handel is covered in leather strips wich deyed a black color.
```
Comment (melissa-kun-li) [inline Assets/Scenes/Level2.unity:558]: awesome! just a few typos:
"The handle is covered in leather strips which are dyed black."

File: Assets/Scenes/Level2.unity [hunk 2]
```diff
@@ -554,14 +554,14 @@ MonoBehaviour:
...
+  brushMessage: The sword has bees cleand in advance.
```
Comment (melissa-kun-li) [inline Assets/Scenes/Level2.unity:559]: "The sword has been cleaned in advance."

File: Assets/Scenes/Level2.unity [hunk 3]
```diff
@@ -554,14 +554,14 @@ MonoBehaviour:
...
+  materialMessage: 70% Carbon 20% Hydrogen 9% Oxygen
+  coveredMessage: The sword has bees cleand in advance.
```
Comment (melissa-kun-li) [inline Assets/Scenes/Level2.unity:561]: "The sword has been cleaned in advance."

File: Assets/Scenes/Level2.unity (hunk 2 above)
Comment (OZoneGuy) [reply Assets/Scenes/Level2.unity:559]: 🐝 🐝 🐝 🐝 🐝

File: Assets/Scenes/Level2.unity [hunk 4]
```diff
@@ -2543,14 +2876,14 @@ MonoBehaviour:
...
-  noneMessage: 
-  magniferMessage: 
-  brushMessage: 
-  materialMessage: 
-  coveredMessage: 
-  magnifierClip: {fileID: 0}
-  materialClip: {fileID: 0}
-  tipManager: {fileID: 0}
+  noneMessage: NA
+  magniferMessage: Overall A simple design, but is not devoid of some decoration.
```
Comment (melissa-kun-li) [inline Assets/Scenes/Level2.unity]: "The design is simple and has few decorations."
or something like that

Comment (OZoneGuy) [reply Assets/Scenes/Level2.unity]: Any particular reason for the rewording? Or just to shorten it?

**Pull request merged**